packages/editor-ui/src/plugins/i18n/locales/en.json

```
@@ -1095,5 +1094,97 @@
 			"missingWorkflow": "Missing workflow",
 			"arrayEmpty": "Must have an array of nodes"
 		}
-	}
+	},
+	"404_NOT_FOUND_MESSAGE": "Oops, couldn’t find that",
```

Not sure if the error code belongs in a key.
```suggestion
	"URL_NOT_FOUND_MESSAGE": "Oops, couldn’t find that",
```